packages/elements/src/react/common/index.ts

```diff
@@ -1,2 +1 @@
-export { Field, FieldError, FieldState, Form, GlobalError, Input, Label, Submit } from '~/react/common/form';
-export { SocialProviderIcon } from '~/react/common/third-party-providers/social-provider';
```

Comment:
Removes `SocialProviderIcon` as it's useless outside of the `<SocialProvider>` context, so to remove confusion it's not exported here anymore